Library/Homebrew/tap_constants.rb

```diff
@@ -2,21 +2,29 @@
 # frozen_string_literal: true
 
 # Match taps' formulae, e.g. `someuser/sometap/someformula`
-HOMEBREW_TAP_FORMULA_REGEX = T.let(%r{^([\w-]+)/([\w-]+)/([\w+-.@]+)$}, Regexp)
+HOMEBREW_TAP_FORMULA_REGEX = T.let(%r{^(?<user>[\w-]+)/(?<repo>[\w-]+)/(?<name>[\w+-.@]+)$}, Regexp)
+# Match default formula taps' formulae, e.g. `homebrew/core/someformula` or `someformula`
+HOMEBREW_DEFAULT_TAP_FORMULA_REGEX = T.let(
+  %r{^(?:[Hh]omebrew/(?:homebrew-)?core/)?(?<name>[a-z0-9\-_]+(?:@[a-z0-9\-_.]+)?)$},
+  Regexp,
+)
```

Comment:
This regex is not accurate, e.g. `homebrew/core/z.lua` doesn't match.

Is it intentional you've copied the cask regex rather than the formula one used above or is it just a copy-paste error?

Reply:
Just a copy-paste error.